parquet-arrow/src/main/java/org/apache/parquet/arrow/schema/SchemaConverter.java

```diff
@@ -471,6 +469,7 @@ public TypeMapping convertINT64(PrimitiveTypeName primitiveTypeName) throws Runt
           case LIST:
           case MAP:
           case MAP_KEY_VALUE:
+          case TIME_MILLIS:
             throw new IllegalArgumentException("illegal type " + type);
         }
```

Comment:
And I changed above in SchemaConverter#convertINT64 because TIME_MILLIS only takes int32.

Reply:
@julienledem @xhochy Could you please check this?

Reply:
@xhochy I would like this too, please